cms/djangoapps/contentstore/views/course.py

```diff
@@ -283,6 +294,28 @@ def course_rerun_handler(request, course_key_string):
             })
 
 
+@login_required
+@ensure_csrf_cookie
+@require_GET
```

Comment:
@chrisndodge I'm okay with this so that the PR can land, but we should ensure that all future work uses the approved best practices rather than relying on the existing patterns.